docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/managing-channels.md

```diff
@@ -1,6 +1,6 @@
 # Managing Channels
```

Comment:
This provides a step-by-step instruction on how to create a channel, but doesn't explain what the configurations do. We need to explain what they're doing.